wikidatarefisland/data_model/filters.py

```diff
@@ -0,0 +1,30 @@
+class StatementFilters:
+    @staticmethod
+    def referenced_statement_excluder(statement):
+        """
+        Filter that returns false for all statements that have any references
+        """
+        return "references" not in statement
```

Comment:
I think in the dumps, references always exist but it's empty array.

Reply:
Cool; this sounds like a big awful bug should I address it here or open a ticket?

Reply:
Yes please, Otherwise I'm worried it wouldn't output anything.

Reply:
So I built this with the actual prod dump; it does output statements but I'll make a ticket to investigate situations where the dump format has empty (but existing) reference blocks in case we are missing some :)